app/scripts/controllers/transactions/index.js

```diff
@@ -347,13 +343,12 @@ export default class TransactionController extends EventEmitter {
     const originalTxMeta = this.txStateManager.getTx(originalTxId)
     const { txParams } = originalTxMeta
     const lastGasPrice = gasPrice || originalTxMeta.txParams.gasPrice
-    const suggestedGasPriceBN = new ethUtil.BN(ethUtil.stripHexPrefix(this.getGasPrice()), 16)
```

Comment:
Could you swap this out for `query.gasPrice()` instead? I think it is important to get an updated gas estimate here for retries. Support has encountered issues in the past with users getting stuck on failed transactions, where a 1.1x increase in price still required _a lot_ of retries before it got high enough to not be rejected outright by the network.

Reply:
I realize that you called this out already in the OP, but it seemed like it'd be easier to include it now, and preserve the existing logic rather than have to resurrect it.

Reply:
Wait a minute... 🤔 What is it doing with that `gasPrice` parameter!? That doesn't seem right.

Reply:
I see - cool, we're bumping it by 1.1x twice - once here, once on the front end. Sure, why not.

On second thought maybe you were right to hold off on this - this is pretty wonky.